### ISSUE TITLE
rotation in volumetric.py

### ISSUE THREAD

**Issue by qiuzhongwei-USTB** [karfly/learnable-triangulation-pytorch#50]
# random rotation
            if self.training:
                theta = np.random.uniform(0.0, 2 * np.pi)
            else:
                theta = 0.0

            if self.kind == "coco":
                axis = [0, 1, 0]  # y axis
            elif self.kind == "mpii":
                axis = [0, 0, 1]  # z axis

            center = torch.from_numpy(base_point).type(torch.float).to(device)

            # rotate
            coord_volume = coord_volume - center
            coord_volume = volumetric.rotate_coord_volume(coord_volume, theta, axis)
            coord_volume = coord_volume + center

hi, as shown in above codes, why you random rotate the coord_volume ?

**Comment by lambdaloop** [comment]
I'm guessing this is for the training augmentation. 
From the paper:
> We did not apply any augmentations during the training, other than rotating the orientation of the cube in volumetric triangulation around the vertical axis.

**Comment by yurymalkov** [comment]
@lambdaloop Yes, this is correct.
